WTWSMS/history/titles/c_cordoba.txt

```diff
@@ -9,14 +9,13 @@
 510.1.1={
 	holder=8600
 }
-554.1.1={
-	holder=36553 # Opimius Gaius Comenciolus 
-	liege = e_byzantium
-}
-572.1.1={ 
+550.1.1={ 
 	holder=36558 # Flavius Constans    
 	liege = 0
 }
+572.1.1={
+    holder=212873
+}
 586.1.1={ 
 	holder=36556 #Reccared
 }
```

Comment:
Why is Cordoba separate from Aracena?

Reply:
I insist. This fictional character was a rebel in Corduba. So this obviously means he held Corduba, and nothing more.

Reply:
This is Corduba, not Aracena